akka-http/src/main/scala/akka/http/javadsl/server/RequestContext.scala

```diff
@@ -40,15 +48,24 @@ class RequestContext private (val delegate: scaladsl.server.RequestContext) {
     log:              LoggingAdapter,
     settings:         RoutingSettings): RequestContext = wrap(delegate.reconfigure(executionContext, materializer, log, settings.asScala))
 
-  def complete[T](value: T, marshaller: Marshaller[T, HttpResponse]): CompletionStage[RouteResult] = {
+  def completeWith[T](value: T, marshaller: Marshaller[T, HttpResponse]): CompletionStage[RouteResult] = {
```

Comment:
yea. I would break name convention with `scaladsl.RequestContext` if this change remain